parm/use_cases/model_applications/convection_allowing_models/read_ascii_storm.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import print_function
```

Comment:
Testing leaving an inline comment. This file should not actually be included in the Point2Grid wrapper pull request. It belongs in feature_406